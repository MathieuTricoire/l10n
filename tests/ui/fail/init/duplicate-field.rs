l10n::init!({
    functions: {},
    functions: {}
});

fn main() {}
