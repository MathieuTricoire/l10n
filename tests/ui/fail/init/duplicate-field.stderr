error: duplicate `functions` field
 --> tests/ui/fail/init/duplicate-field.rs:3:5
  |
3 |     functions: {}
  |     ^^^^^^^^^
